src/diffusers/models/vae.py

```diff
@@ -401,10 +403,13 @@ def __init__(
         up_block_types: Tuple[str] = ("UpDecoderBlock2D",),
         block_out_channels: Tuple[int] = (64,),
         layers_per_block: int = 1,
+        decoder_layers_per_block: int = 1,
```

Comment:
This is currently breaking - we should instead do: 
```suggestion
        decoder_layers_per_block: Optional[int] = None,
```

And then if it's None overwrite with `layers_per_block`